episodes/introduction.md

```diff
@@ -1,28 +1,246 @@
+
+<!-- Throughout the process, you may wish to get feedback from your peers and mentors about your progress and approach. -->
+<!-- There are many steps you can take to make collaboration easier technologically as well as ensuring your work is legible to others. -->
+
+The parts of the research project life cycle that typically receive explicit attention in coursework include selecting a research question, analysis techniques (e.g., selecting the appropriate statistical method), and publication.
+Training in the remaining portions is often assumed to happen naturally through research experience, but this is not always effective in practice.
+Gaps in these areas can create roadblocks to conducting effective, reproducible and open research, even for experienced researchers.
+This course aims to address some of those roadblocks.
+
+:::::::: challenge
+
+#### Introspection: untaught skills
+
+Think of a skill you've found useful in your research that wasn't taught in one of your classes. How did you learn it? Have you taught it to anyone else? Why or why not?
+
+::::::::
+
+#### What we will cover
+
+This course is focused on practical solutions to roadblocks you may encounter in dealing with data, code, and collaboration. We will be following the arc of an open data analysis project in Python, structuring the course into three sections:
+
+Roadblocks to data acquisition:
+- My data is in a format I've never worked before
+- The data I want to work with is published through an Application Programming Interface
+(API), and I don't know how to download it
+
+Roadblocks to data cleaning & processing:
+- There's something unexpected about my input data, but I'm not sure what
+- The code runs on some of my data, but errors on other input data
+- When I re-run my code I get different results, and I'm not sure why
+- I have no idea which part of my code is causing a particular problem
+- My data changes format or content over time
+- My data is too big to work with on a desktop computer
+
+Roadblocks to collaboration:
+- I'm not sure how to make it simple for collaborators to run and contribute to my code
+- I need to publish my code and/or data for a paper I'm submitting to, but I'm not sure
+how best to do so
+- A colleague wants to build on my existing code, but I'm not sure how to clearly document
+what I've done to make it possible for them to adapt it
+- I wrote this code myself six months ago, and I do not recognize it, nor can I remember what I was trying to do
+
+The next episode will discuss reading data in unexpected file formats.
+
+:::::::: instructor
+
+The remaining section is useful, but not central, and can be cut for time if necessary. If cut, direct students to the course website for tips on locating and identifying appropriate datasets for open research projects.
```

Comment:
I made "finding data" optional and cut the objective, since it's the most conducive to offline reference if we need to cut for time